scopes/react/react-native/webpack/webpack-transformers.ts

```diff
@@ -1,17 +1,31 @@
 import { WebpackConfigTransformer, WebpackConfigMutator, WebpackConfigTransformContext } from '@teambit/webpack';
 
+const reactNativeElementRule = {
+  test: /\.(jsx?|tsx?)$/,
+  include: [/node_modules\/react-native-/],
```

Comment:
Why `react-native-`? It should be `react-native` or I'm wrong?

Reply:
No, this is for any react-native-<some-package> package, e.g. react-native-paper, that's in node_modules